src/Ombi/ClientApp/src/app/requests-list/components/movies-grid/movies-grid.component.scss

```
@@ -1,49 +1,457 @@
+// Card Grid
+.card-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
+  gap: 20px;
+  animation: fadeUp 0.3s ease;
 }
 
-::ng-deep .row {
-	margin-right:0;
-	margin-left:0;
+@keyframes fadeUp {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)